docs/software/python.md

```diff
@@ -1,44 +1,40 @@
 # Python and R with Anaconda
 
 To support diverse _Python_ and _R_ workflows, Research Computing users can utilize [Anaconda](http://anaconda.com). Anaconda 
-provides the _conda_ package manager, which allows for easy installation of software and associated 
-packages. The _conda_ package manager provides support for _Python_, _R_, and many other application stacks. 
+provides the _Conda_ package manager, which allows for easy installation of software and associated 
+packages. The _Conda_ package manager provides support for _Python_, _R_, and many other application stacks. 
 CURC also supports [Mamba](https://mamba.readthedocs.io/), an alternative package manager that allows parallel downloading of repository data and package files using multi-threading.
 
 The following documentation describes how to activate the CURC Anaconda distribution and our default environments, as well as how to create and activate your own custom Anaconda environments. For more information on utilizing [Mamba](https://mamba.readthedocs.io/) (an alternative package manager) please see the section [Mamba Package Manager](#mamba-package-manager) below. Additional documentation on [CURC OpenOnDemand](../open_ondemand/jupyter_session.md) is available for users desiring to interact with their custom environments via [Jupyter notebooks](https://jupyter.org). 
 
-## Configuring conda with `.condarc`
+## Configuring Conda and Mamba with `.condarc`
 
-The conda package manager allows modification of default settings to be done through a text file known as the `.condarc`. This file exists within a user's `/home/$USER/` directory and can be quickly be accessed using the file's full path at `~/.condarc`.
+The Conda and Mamba package managers allow modification of default settings. These settings are specified in a text file called `.condarc`. For possible configuration options, please see [Using the .condarc conda configuration file](https://docs.conda.io/projects/conda/en/latest/user-guide/configuration/use-condarc.html#using-the-condarc-conda-configuration-file). The `.condarc` file should exist within a user's `/home/$USER` directory and can be quickly accessed using the file's full path at `~/.condarc`. A `.condarc` file is important because by default Conda and Mamba will put all package source code and environments in your `/home/$USER` directory. This quickly becomes an issue because your `/home/$USER` directory only has a capacity of 2 GB and packages can be several gigabytes in size.
```

Comment:
On Line 12 - "This quickly becomes an issue because your /home/$USER directory only has a capacity of 2 GB and packages can be several gigabytes in size."

Should we remove the "2 GB" to future-proof this section in case the /home directory ends up being increased?

Like: "This quickly becomes an issue because your /home/$USER directory has limited storage capacity which cannot adequately support large packages, which can be several gigabytes in size."

Reply:
Great suggestion! I will go ahead and make that change.